### ISSUE TITLE
add cellrank krylov optimised packages back in

### ISSUE THREAD

**Issue by le-ander** [theislab/extended-single-cell-best-practices-container#2]

**Comment by Zethson** [comment]
Do you have an update here?

**Comment by le-ander** [comment]
No, getting this to run is rather tricky without conda. I'd day let's see if it runs at acceptable speed without these packages first yor your usecase.

**Comment by Zethson** [comment]
All right. FYI @WeilerP @AnnaChristina

**Comment by WeilerP** [comment]
Could work depending on the size of the dataset. For the tutorials using the `VelocityKernel` and `CytoTRACEKernel`, we're only using the vanilla CellRank installation, _i.e._, no `petsc` or `slepc`.
